### ISSUE TITLE
Added diporder module - [merged]

### ISSUE THREAD

**Issue by hejamu** [maicos-devel/maicos#148]
In GitLab by @PicoCentauri on Mar 15, 2019, 16:12 UTC

_Merges diporder -> develop_

This PR added the diporder module. 

@aschlaich suggested to also merge the higher order moment order parameters from his LandauGinzburg repo. I looked quickly at the scripts but as we already discussed I also see currently no easy (or better nice) way to combine them. The quick and dirty way would just the squeeze them in and decide by an if clause... Furthermore they are hardcoded to three point watermodels. I highly recommend that we generalise this before merging?!
TL;DR Merge higher order moment order parameters later

If there are no other things in this PR I only have to adjust the bash completion script before merging.

*Migrated from GitLab: https://gitlab.com/maicos-devel/maicos/-/merge_requests/28*

**Comment by hejamu** [comment]
In GitLab by @PicoCentauri on Mar 29, 2019, 13:44 UTC

added 3 commits

<ul><li>0a29a808...740c49a1 - 2 commits from branch <code>ag-netz:develop</code></li><li>363b9328 - Added diporder module</li></ul>

[Compare with previous version](https://gitlabph.physik.fu-berlin.de/ag-netz/mdtools/merge_requests/28/diffs?diff_id=371&start_sha=0a29a808424dab5fb0c36bbe0baa7b39f89dc2af)

 *By Philip Loche on 2019-03-29T13:44:02 (imported from GitLab project)*

**Comment by hejamu** [comment]
In GitLab by @PicoCentauri on May 8, 2019, 09:26 UTC

merged

 *By Philip Loche on 2019-05-08T09:26:29 (imported from GitLab project)*
